core/EE_Addon.core.php

```diff
@@ -810,50 +815,12 @@ public function after_plugin_row($plugin_file, $plugin_data, $status)
                 $after_plugin_row .= '<tr id="' . sanitize_title($plugin_file) . '-ee-addon" class="' . $class . '">';
                 $after_plugin_row .= '<th class="check-column" scope="row"></th>';
                 $after_plugin_row .= '<td class="ee-addon-upsell-info-title-td plugin-title column-primary">';
-                $after_plugin_row .= '<style>
-.ee-button,
-.ee-button:active,
-.ee-button:visited {
-	box-sizing: border-box;
-	display: inline-block;
```

Comment:
Moved these styles to `core/admin/assets/admin-menu-styles.css` file. I don't know if it is the right file or not but based on my checks it was loading on the plugins page.

Reply:
ya that will be fine for now... I think in the dev branch there is a better place, but it's not a  big deal at all